functions/supplier/mongodb-supplier/src/main/java/org/springframework/cloud/fn/supplier/mongo/UpdatingMongoDbMessageSource.java

```diff
@@ -1,118 +0,0 @@
-/*
- * Copyright 2021-2021 the original author or authors.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *      https://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package org.springframework.cloud.fn.supplier.mongo;
-
-import java.util.Collections;
-import java.util.List;
-
-import org.springframework.data.mongodb.core.MongoTemplate;
-import org.springframework.data.mongodb.core.query.BasicQuery;
-import org.springframework.data.mongodb.core.query.BasicUpdate;
-import org.springframework.data.mongodb.core.query.Query;
-import org.springframework.data.mongodb.core.query.Update;
-import org.springframework.expression.EvaluationContext;
-import org.springframework.expression.Expression;
-import org.springframework.expression.TypeLocator;
-import org.springframework.expression.spel.support.StandardTypeLocator;
-import org.springframework.integration.endpoint.AbstractMessageSource;
-import org.springframework.integration.expression.ExpressionUtils;
-import org.springframework.integration.mongodb.inbound.MongoDbMessageSource;
-import org.springframework.integration.mongodb.support.MongoHeaders;
-import org.springframework.lang.Nullable;
-import org.springframework.messaging.Message;
-import org.springframework.util.Assert;
-
-/**
- * An {@link AbstractMessageSource} extension for MongoDB updates
- * based on the query result from the {@link MongoDbMessageSource} delegate.
- *
- * @author Artem Bilan
- */
-class UpdatingMongoDbMessageSource extends AbstractMessageSource<Object> {
```

Comment:
Why was this deleted?

Reply:
I added that in the commit message

Reply:
DOH! Yes you did. My bad. 

> Remove UpdatingMongoDbMessageSource since MongoDbMessageSource
supports now an update option in Spring Integration

I did read the description (but forgot it all by the time  I got all the way to this file). I should have went back to the description before commenting.